quidel_covidtest/tests/test_run.py

```diff
@@ -68,6 +68,15 @@ def test_output_files(self, clean_receiving_dir):
         )
         assert (df.columns.values == ["geo_id", "val", "se", "sample_size"]).all()
 
+        df = pd.read_csv(join("./receiving", "20200718_county_covid_ag_raw_pct_positive.csv"))
+        # TODO: should megacounty 51000 be here (it currently is)?
+        assert set(df.geo_id) == set([51143, 51059, 51000])
```

Comment:
@jingjtang I wanted to add some tests for the functionality you mentioned in your last comment.

> 
> As for raw signals:
> 
> For counts smaller than 50, we report nothing.
> For counts larger than 50, we report what we have.
> The counts for raw signals are just the counts for the corresponding date (different from the smoothed signals)

`df` here is

```
   geo_id        val        se  sample_size
0   51000  51.923077  5.693813         77.0
1   51059  40.163934  6.328838         60.0
2   51143  50.000000  6.250000         64.0
```

- ✔️ZIP 24534, FIPS 51083 has 4 counts, so it's below threshold and absent.
- ✔️ZIP 24529, FIPS 51117 has 24 counts, so it's below threshold and absent.
- ✔️ZIP 24526, FIPS 51019 has 49 counts, so it's below threshold and absent.
- ✔️MEGAFIPS 51000 has 77 = 49 + 24 + 4 counts, so it aggregates the two above.
- ✔️ZIP 24527, FIPS 51143 has 64 counts, so it's above threshold and present.
- ✔️ZIP 22079, FIPS 51059 has 60 counts, so it's above threshold and present.

I guess, I have no question here. This is mostly a comment to make sure I understood this, so feel free to ignore (we can remove that #TODO).